src/commcare_cloud/ansible/roles/citusdb/tasks/setup_worker.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
is it necessary to create users on the workers? I'm not clear how citus communicates with the workers. Also wondering if there needs to be provision in the pg_hba.conf file.

Reply:
I think these users are only used by the application, Citus should have it's own way of communicating. 
But Since we have not installed pgbouncer on workers, I think application never interacts with worker and thus these users are not very useful in that sense. 
From provision in the pg_hba.conf , do you mean adding configurations so coordinator can connect to the worker or allowing these users to connect.

Reply:
Allowing coordinator to connect to workers. In the tests I did I don't recall having to change anything but just wanted to make sure.